OpenRA.Mods.Common/Traits/Repairable.cs

```diff
@@ -64,7 +64,7 @@ public IEnumerable<IOrderTargeter> Orders
 		{
 			get
 			{
-				yield return new EnterAlliedActorTargeter<BuildingInfo>("Repair", 5, CanRepairAt, _ => CanRepair() || CanRearm());
+				yield return new EnterAlliedActorTargeter<BuildingInfo>("Repair", 5, (a, m) => CanRepairAt(a), _ => CanRepair() || CanRearm());
```

Comment:
This change should be dropped to avoid regressing #16607

Reply:
Reverted.